libs/zzz/consts/src/wengine.ts

```diff
@@ -67,6 +67,10 @@ export const allWengineKeys = [
 
 export type WengineKey = (typeof allWengineKeys)[number]
 
+export function isWengineKey(key: unknown): key is WengineKey {
+  return typeof key === 'string' && allWengineKeys.includes(key as WengineKey)
+}
+
 export const allModificationKeys = [0, 1, 2, 3, 4, 5] as const
```

Comment:
modification

Reply:
> modification

is this a joke I'm too dyslexic to understand?

Reply:
oh i thought this was supposed to be replaced with milestonekey

Reply:
it's not used, I'll remove it